umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/umh-lite.md

```diff
@@ -0,0 +1,33 @@
+---
+title: UMH Lite
+menuTitle: UMH Lite
+content_type: feature
+description: |
+    Introducing UMH Lite - a new type of UMH installation.
+weight: 5000
+---
+
+If you are already using Unified Namespace, or have a Kafka / MQTT broker; and want to try out the basic UMH features, you can go for the UMH Lite installation.
+
+If you want the full-featured UMH experience, we recommend you to install the Classic version. You can see a comparison of the features between the two versions below.
+
+### UMH Classic vs UMH Lite
```

Comment:
You can use these feature names here: 
![Screenshot 2024-04-30 at 10 16 12](https://github.com/united-manufacturing-hub/umh.docs.umh.app/assets/30755722/9a095f67-f4f7-4a18-bb66-713c2f9e317e)

Reply:
Unfortunately markdown does not supported nested tables. Will see what I can do.

Reply:
Updated with raw HTML.